src/System.Security.Cryptography.ProtectedData/src/System.Security.Cryptography.ProtectedData.builds

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.Security.Cryptography.ProtectedData.csproj" />
```

Comment:
I don’t know what a .builds file for a test assembly would look like. There are no existing .builds file for any test assemblies,
and all of the recursive *.builds patterns I can see in the proj files look only in the pkg and src directories.